src/protocol/templates/pingmessage.py.in

```diff
@@ -41,24 +42,27 @@ payloadDict = {
 {{structToken[field.type]}}
 {%- endif %}
 {% endfor %}",
-                       "field_names": (
+        "field_names": (
```

Comment:
I do believe we should keep this the way that it is for readability: https://github.com/bluerobotics/ping-python/blob/master/Ping/PingMessage.py#L59

Reply:
don't you think this is better?
https://github.com/bluerobotics/ping-python/pull/22/files#diff-a006c457cee7837b799e661206f71524R59

Reply:
Yes, I'm sorry.

Reply:
a case of me trying to process the template in my head - lazy